extensions/wikia/Blogs/BlogsHelper.class.php

```diff
@@ -172,7 +174,13 @@ public static function onAbortMove( Title $sourceTitle, Title $targetTitle, User
 		return true;
 	}
 
-	public static function onPageHeaderActionButtonShouldDisplay( \Title $title, &$shouldDisplayActionButton ) {
+	/**
+	 * @param Title $title
+	 * @param bool $shouldDisplayActionButton
+	 *
+	 * @return bool
+	 */
+	public static function onPageHeaderActionButtonShouldDisplay( \Title $title, bool &$shouldDisplayActionButton ): bool {
 		if ( $title->getNamespace() === NS_BLOG_LISTING ) {
 			$shouldDisplayActionButton = false;
 		} else if ( $title->getNamespace() === NS_BLOG_ARTICLE && !$title->userCan( 'edit' ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This branch's code block is the same as the block for the branch on line 184. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)